### DISCUSSION TITLE
Prevent output of markdown file

### DISCUSSION THREAD

**Discussion by coderaiser** [remarkjs/remark#535] [Q&A] [answered]
Could you please tell me what flags should I use to prevent showing content of markdown file:

     remark README.md -q

Doesn't works for me. What I need is to show only errors and warnings.

**Comment by wooorm** [comment, marked as answer]
[`--no-stdout`](https://github.com/unifiedjs/unified-args/blob/main/readme.md#--stdout)!

**Reply by coderaiser** [reply]
Thanks a lot :), this works good.
